src/apprt/gtk/builder_check.zig

```diff
@@ -0,0 +1,43 @@
+const std = @import("std");
+const build_options = @import("build_options");
+
+pub const c = @cImport({
+    @cInclude("gtk/gtk.h");
+    if (build_options.adwaita) {
+        @cInclude("libadwaita-1/adwaita.h");
```

Comment:
I don't understand why Zig is forcing us to include the `libadwaita-1` part of the path here. In C, its `adwaita.h`

Reply:
Must be something left over from previous eras because it no longer seems to be necessary, I was just cargo-culting what was in `gtk/c.zig`.